examples/gke-storage-parallelstore.yaml

```diff
@@ -76,32 +76,46 @@ deployment_groups:
     settings:
       name: sample-pool
       zones: [$(vars.zone)]
-      machine_type: n2-standard-4
+      machine_type: n2-standard-16
 
-  ### Parallelstore enabled Job ###
+  # Train a TensorFlow model with Keras and Parallelstore on GKE
+  # Tutorial: https://cloud.google.com/parallelstore/docs/tensorflow-sample
 
   - id: parallelstore-job
```

Comment:
nit: can you please add which model/training this job will run  ? Or add a comment

Reply:
it is already mentioned in line 81, one line above

Reply:
Ack , oops!